thunder/torch/default_torch_ops.py

```diff
@@ -756,3 +756,28 @@
         torch.fft.rfftn,
     ],
 }
+
+_auto_registered_operators_returning_views = [
```

Comment:
A comment here, like what this list does, what it means, and how it's generated, would be helpful

Reply:
Hi @mruberry , the list of ops that return or may return tensor views are used in the functionalization, I pointed it to the in-place check function that uses this list